src/EnergyPlus/SimulationManager.cc

```diff
@@ -1609,16 +1609,15 @@ namespace SimulationManager {
                                  "file specified; but no weather file specified.");
             }
         }
-        if (!state.dataGlobal->DoDesDaySim && !state.dataGlobal->DoWeathSim) {
+
+        if (!state.dataGlobal->DoDesDaySim && !state.dataGlobal->DoWeathSim && !state.dataGlobal->DoHVACSizingSimulation) {
```

Comment:
This change seems reasonable, we just need to exit if we aren't doing _anything_.  I feel I should ping @EnergyArchmage for a quick thumbs up or down...

Reply:
Well, I believe the issue 9191 is more than just fataling gracefully. What is desired is to be able to run just ideal loads sizing calcs with no full simulation of design days or run periods. And get component loads and equipment sizing results. It is big performance issue. 

I happen to also have been working on this just today and have it working. If you are interested in a more expansive interpretation of 9191 I will check if I can share.

Reply:
I think it would be valuable if you are able to share it.  Mostly from a perspective of you not having to manage conflicting changes.  If you aren't able to share, I'd be happy to massage our changes to maximize their value in addressing 9191 and reducing the burden on your fork.  Let me know when you find out so we can proceed either way, and thanks!